C/include/simplicity/errorCodes.h

```diff
@@ -67,6 +66,8 @@ static inline const char * SIMPLICITY_ERR_MSG(simplicity_err err) {
     return "Program has STOP node";
   case SIMPLICITY_ERR_HIDDEN:
     return "Program has illegal HIDDEN child node";
+  case SIMPLICITY_ERR_BITSTREAM_EOF:
+    return "Unexpected end of bitstream";
   case SIMPLICITY_ERR_BITSTREAM_TRAILING_BYTES:
```

Comment:
If we unify the bitstream and witness error variants, then we have to update the error messages.

"Trailing bytes after final byte of witness" → "Trailing bytes after final byte of encoded object" (or something similar)

Reply:
What's wrong with `witness`?  Would you be happy with `Simplicity witness`?

Reply:
If we keep program and witness error variants separate, then I'm happy with the error messages as they currently are.